src/StreamWrapper.php

```diff
@@ -106,6 +106,13 @@ public function stream_seek($offset, $whence)
         return true;
     }
 
+    public function stream_cast($cast_as)
+    {
+        $stream = clone($this->stream);
```

Comment:
Why the brackets?

Reply:
There is no difference. Usually it's better to use them to avoid some issues.
Do you want me to remove them?